### PR TITLE
INGM-405: New example of using PDO poller

### PR DESCRIPTION
**Pull request by WoodElv**
### Description

The example establish a CoE communication, set a type of sensor for position and velocity feedback sensors, and read them using a poller with PDO.

### Type of change

- [x] Example using PDO poller.

### Tests
- [x] Unit test that checks all functions are called.

Please describe the tests that you run to verify your changes if it applies. 

### Documentation

Please update the documentation.

- [x] Use [type hints](https://docs.python.org/3/library/typing.html) for every function and argument.
- [x] Build documentation locally to verify changes.

### Code formatting

- [x] Use black package to format the code: `black -l 100 ingeniamotion tests`. It is recommended to configure the code editor to automatically format the code using black with a max length line of 100.

### CODE REVIEW

File: examples/pdo_poller_example.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+def set_feedback_sensors(mc: MotionController) -> None:
+    """Set the type of position and velocity feedback sensors.
+
+    Args:
+        mc: the controller to configure.
+    """
+    # Modify the SensorType.
+    mc.configuration.set_position_feedback(SensorType.HALLS)
+    mc.configuration.set_velocity_feedback(SensorType.HALLS)
```
Comment (GullinBustin) [inline examples/pdo_poller_example.py:43]: Why do you set HALLS sensor position and velocity feedbacks? This is the feedback with the worst resolution, it is an unusual case.

File: examples/pdo_poller_example.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import time
+from typing import Dict, List, Union
+
+from ingeniamotion.enums import SensorType
+from ingeniamotion.motion_controller import MotionController
+
+
+def establish_coe_connection(mc: MotionController) -> None:
```
Comment (GullinBustin) [inline examples/pdo_poller_example.py:8]: Same as baudrate example, you can use our functions directly. Remember that exist these functions:

- connect_servo_ethercat_interface_index: https://distext.ingeniamc.com/doc/ingeniamotion/0.7.1/ingeniamotion/communication.html#ingeniamotion.communication.Communication.connect_servo_ethercat_interface_index
- connect_servo_ethercat_interface_ip: https://distext.ingeniamc.com/doc/ingeniamotion/0.7.1/ingeniamotion/communication.html#ingeniamotion.communication.Communication.connect_servo_ethercat_interface_ip

**Review by GullinBustin — CHANGES_REQUESTED** [review]

File: examples/pdo_poller_example.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+    Args:
+        mc: the controller to configure.
+    """
+    # Modify the SensorType.
+    mc.configuration.set_position_feedback(SensorType.HALLS)
+    mc.configuration.set_velocity_feedback(SensorType.HALLS)
+
+
+def perform_pdo_poller(mc: MotionController) -> None:
```
Comment (mjedieckmann) [inline examples/pdo_poller_example.py:46]: "set_up" instead of "perform" sounds better

**Review by mjedieckmann — APPROVED** [review]

**Review by GullinBustin — APPROVED** [review]

File: examples/pdo_poller_example.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
+import time
+from typing import Dict, List, Union
+
+from ingeniamotion.enums import SensorType
+from ingeniamotion.motion_controller import MotionController
+
+
+def set_feedback_sensors(mc: MotionController) -> None:
```
Comment (MartinALL91) [inline examples/pdo_poller_example.py:8]: I don't think this is necessary.

File: examples/pdo_poller_example.py [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+    print(f"Time: {time_stamps}")
+    print(f"Actual Position values: {data[0]}")
+    print(f"Actual Velocity values: {data[1]}")
+
+
+def main() -> None:
+    mc = MotionController()
+    # Modify these parameters to connect a drive
+    interface_index = 3
```
Comment (MartinALL91) [inline examples/pdo_poller_example.py:52]: I think its better if we use the interface IP address

**Review by MartinALL91 — APPROVED** [review]

**Pull request merged**